docs/labwc-config.5.scd

```diff
@@ -890,6 +890,24 @@ situation.
 	can be caused by *<margin>* settings or exclusive layer-shell clients
 	such as panels.
 
+## MENU
+
+```
+<menu>
+  <ignoreButtonReleasePeriod>200</ignoreButtonReleasePeriod>
```

Comment:
This is still stating `200` ms, same for the `rc.xml.all` snippet. Sorry, missed that in the earlier review.
With that fixed this LGTM so feel free to Rebase and Merge. Thanks!

Reply:
Done. Thank you!